src/script.js

```diff
@@ -0,0 +1,54 @@
+const days = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+
+function createCalendar(id, year, month) {
+    const currentMonth = month - 1;
+    const date = new Date(year, currentMonth);
+    const tableDiv = document.querySelector(`#${id}`);
+    const table = document.createElement("table");
+    let tr = document.createElement("tr");
+
+    days.forEach((day) => {
+        const th = document.createElement("th");
+        th.innerText = day;
+        tr.appendChild(th);
+    });
+    table.appendChild(tr);
+
+    tr = document.createElement("tr");
+    for (let i = 0; i < getDay(date); i++) {
+        tr.appendChild(document.createElement("td"));
+    }
+
+    while (date.getMonth() === currentMonth) {
+        const td = document.createElement("td");
+        td.innerHTML = date.getDate();
+        tr.appendChild(td);
+
+        if (getDay(date) % 7 === 6) {
+            table.appendChild(tr);
+            tr = document.createElement("tr");
+        }
+
+        date.setDate(date.getDate() + 1);
```

Comment:
will be better to use own state instead of use date object or calculate counts before and use for loop instead of while